src/stackdriver-nozzle/mocks/heartbeater.go

```diff
@@ -0,0 +1,27 @@
+package mocks
```

Comment:
this seems like a legit version of the heartbeater, maybe we should just use the real version in tests instead of a mock?